setup.py

```diff
@@ -2,7 +2,7 @@
 
 setup(
     name = 'xpdacq',
-    version = '1.0.2',
```

Comment:
I should add: we love using "versioneer," which auto-detects your latest git tag and makes `__version__` for you. Once that's set up, you never have to think about it again. I'll help someone set that up in exchange for more bluesky docs reviews.... ;- )